Add tbody_html and row_html options to TableFor and IndexAsTable (#8423)
activeadmin/activeadmin: docs/3-index-pages/index-as-table.md

end
```

## Custom tbody HTML attributes

> In order to add special html attributes to table tbody pass a `:tbody_html` option of the `index` method.

Reviewer: ```suggestion
In order to add HTML attributes to the tbody use the `:tbody_html` option.
```

```ruby
index tbody_html: { class: "my-class", data: { controller: 'stimulus-controller' } } do
  # columns
end
```

## Custom row HTML attributes

> In order to add special html attributes to table rows pass the proc object as a `:row_html` option

Reviewer: ```suggestion
In order to add HTML attributes to table rows, use a proc object in the `:row_html` option.
```
